server/lib/donations.js

```diff
@@ -120,9 +121,10 @@ export const processDonation = (Sequelize, donation) => {
           { donation: donation.info,
             user,
             group,
-            interval: subscription && subscription.interval
+            interval: subscription && subscription.interval,
             // TODO: bring this back. Figure out how to pass the application link
```

Comment:
Leaving the todo because a better link would be one with a token, so you don't have to relogin.